### DISCUSSION TITLE
Taking the name of an old unused account

### DISCUSSION THREAD

**Discussion by resigned** [community/community#22025] [New to GitHub]
<p>Hey, I head you can take the names of old unused accounts. Does anyone know the proccess in doing so?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi resigned, </p>
<p>Thanks for asking!</p>
<p>Usernames at GitHub are offered on a first-come-first-served basis, and are intended for immediate use. If a name might be available for you to use, you will find that out during the process of <a href="https://help.github.com/en/articles/changing-your-github-username" rel="nofollow noopener">changing your GitHub username</a>.</p>
<p>You may also want to have a read of our <a href="https://help.github.com/articles/name-squatting-policy/" rel="nofollow noopener">Name Squatting Policy</a>.</p>
<p>Hope that helps!</p>

**Comment by resigned** [comment]
<p>Thanks for your response :slight_smile:</p>

**Comment by ghost** [comment]
<p>I got the same flagged status on my account which I just opened.<br>
What does it mean?</p>
<p>Was it ever unflagged?</p>

**Comment by sherif-botros** [comment]
<p>Hello if there is an inactive organisation name, can I get it I am after the organisation named Sherian which is the name of my company</p>

**Comment by sherif-botros** [comment]
<p>Hey there :smiley:  are you the owner of Sherian - <a href="http://github.com/sherian" rel="nofollow noopener">github.com/sherian</a> ?</p>

**Comment by andruskanu** [comment]
<p>Hello, how can I contact an user? I want an username and it’s not avaiable, but the user haven’t any activity and I hope he want to change the username. :smiley:<br>
Thank you!</p>

**Comment by abedshaaban** [comment]
Hello 👋🏼  @shaaban can we chat?

**Comment by lalithtinku** [comment]
i created my account on 2018 and no posts in that acccount. even i forgot my email ID to recover it. how would i know the email id so that i can recover my account.

**Comment by dave-meyer** [comment]
Hello @stomey 👋 Would you be open to chat?
